### DISCUSSION TITLE
Dashboard funktioniert mit 2.8.0 nicht mehr

### DISCUSSION THREAD

**Discussion by chef76** [brutella/hkknx-public#296] [Q&A] [answered]
Seit dem Update auf 2.8.0 funktioniert das Dashboard nicht mehr: 

![image](https://github.com/brutella/hkknx-public/assets/69462419/ed7ed3b9-f4e2-4dd5-b9aa-7ea87225076f)

Jemand eine Idee?

**Comment by brutella** [comment, marked as answer]
Die Funktion [ListenAndServe](https://hochgatterer.me/hkknx/scripts/pkgs/net/http/#ListenAndServe) ist jetzt, wie es eigentlich sein sollte, im Package [net/http](https://hochgatterer.me/hkknx/scripts/pkgs/net/http) verfügbar. 

Du musst also dieses Package verwenden.

```go
var http = import("net/http")
http.ListenAndServe(...)
```

Sorry, das hatte ich wohl übersehen.

**Reply by chef76** [reply]
Die Variable war im Skript schon definiert. Ich musste nur das http. vor die Funktion schreiben.

Läuft wieder, Danke!
